src/resources/views/crud/columns/array_count.blade.php

```diff
@@ -1,24 +1,24 @@
 {{-- enumerate the values in an array  --}}
 @php
     $array = data_get($entry, $column['name']);
-    $suffix = isset($column['suffix']) ? $column['suffix'] : 'items';
+
+    $column['escaped'] = $column['escaped'] ?? false;
+    $column['prefix'] = $column['prefix'] ?? '';
+    $column['suffix'] = $column['suffix'] ?? 'items';
+    $column['text'] = '-';
 
     // the value should be an array wether or not attribute casting is used
     if (! is_array($array)) {
         $array = json_decode($array, true);
     }
 
     if($array && count($array)) {
-        $column['text'] = count($array).' '.$suffix;        
-    } else {
-        $column['text'] = '-';
+        $column['text'] = $column['prefix'].count($array).' '.$column['suffix'];
```

Comment:
I think for consistency we should not make the `suffix` here have a space by default from the column value.

Reply:
Please check above why this space is here 😅